src/index.html

```diff
@@ -3,11 +3,251 @@
+              </li>
+              <li class="mobile-menu__item">
+                <a href="#gallery" class="mobile-menu__link">Gallery</a>
+              </li>
+              <li class="mobile-menu__item">
+                <a href="#consultation" class="mobile-menu__link">Consultation</a>
+              </li>
+              <li class="mobile-menu__item">
+                <a href="#testimonials" class="mobile-menu__link">Testimonials</a>
+              </li>
+              <li class="mobile-menu__item">
+                <a href="#contacts" class="mobile-menu__link">Contacts</a>
+              </li>
+            </ul>
+            <p class="mobile-menu__phone">+1 234 555-55-55</p>
+            <a class="mobile-menu__call" href="tel:+1 234 555-55-55">Call to order</a>
+          </nav>
+        </div>
+
+        <h1 class="header__heading">Miami<br>Condo Kings</h1>
+
+        <nav class="nav header__nav">
+          <ul class="nav__list">
+            <li class="nav__item nav__item--active">Home</li>
+            <li class="nav__item">
+              <a href="#about" class="nav__link">About us</a>
+            </li>
+            <li class="nav__item">
+              <a href="#gallery" class="nav__link">Gallery</a>
+            </li>
+            <li class="nav__item">
+              <a href="#consultation" class="nav__link">Consultation</a>
+            </li>
+            <li class="nav__item">
+              <a href="#testimonials" class="nav__link">Testimonials</a>
+            </li>
+            <li class="nav__item">
+              <a href="#contacts" class="nav__link">Contacts</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <div class="container">
+        <section class="info main__section">
+          <h2 class="info__heading heading" id="about">Miami best<br>&#8233;Real estate <a class="link" href="#">here</a></h2>
+          <div class="info__text content">
+            <p class="content__heading"><a class="content__heading link" href="#">Over 50 years experience in love.</a></p><br>
+            <p>Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p><br>
+            <p>The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists. <br>&#8233;Working directly with developers - so have more useful information about the profitable deals.</p><br>
+            <p><strong>We will never be late!&#8233;<br>Guaranteed secure transaction - 100%</strong></p>
+          </div>
+        </section>
+
+        <section class="gallery main__section">
+          <h2 class="heading section__heading" id="gallery">Gallery<span class="heading__crown"></span> </h2>
```

Comment:
```suggestion
          <h2 class="heading section__heading" id="gallery">Gallery<span class="heading__crown"></span> </h2>
```<h2 class="heading gallery__heading" id="gallery">Gallery<span class="heading__crown"></span> </h2>

Reply:
Повиправляв. Забрав взагалі блок section, який не використовується. Додав модифікатор до heading